docs/src/apps/jsonrpc-api.md

```diff
@@ -635,33 +636,59 @@ curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","id":1, "m
 {"jsonrpc":"2.0","result":{"identity": "2r1F4iWqVcb8M1DbAjQuFpebkQHY9hcVU4WuW2DJBppN"},"id":1}
 ```
 
-### getInflation
```

Comment:
Can you please mark `getInflation` as deprecated in core/src/rpc.rs too, so future readers know why that endpoint is no longer documented

Reply:
Currently, this pr removes the `getInflation` endpoint entirely. My thinking was that there are unlikely to be downstream users, since no inflation on mainnet-beta, and only very new to testnet.
Should I restore?

Reply:
Ah, I totally missed that in the diff.  Yanking it seems fine, I agree